ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-maistra-2.4__servicemesh-ocp4.13-lp-interop.yaml

```
@@ -0,0 +1,67 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  kiali-cypress-tests-runner:
+    name: kiali-cypress-tests
+    namespace: ci
+    tag: v1.57
+  kiali-int-tests-runner:
+    name: kiali-int-tests
+    namespace: ci
+    tag: v1.57
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: maistra-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: servicemesh-interop-aws
```

```suggestion
- as: servicemesh-interop-aws
  remote_api: true
```
This should allow us to [trigger the scenario using the Gangway API](https://cspi-qe.gitbook.io/ocp-ci-onboarding/tutorials/triggering_guide#gangway-api), if needed. You'll also need to run `make update` after this change to update everything properly.